ui/StatusQ/src/StatusQ/Controls/StatusInput.qml

```diff
@@ -403,7 +406,7 @@ Item {
                 Layout.alignment: Qt.AlignVCenter
                 height: visible ? contentHeight : 0
                 visible: root.charLimit > 0
-                text: "%1 / %2".arg(statusBaseInput.text.length).arg(root.charLimit)
+                text: "%1 / %2".arg(Utils.encodeUtf8(statusBaseInput.text).length).arg(root.charLimit)
```

Comment:
Change looks good; my only fear is that it's gonna make the input even slower when typing a (long) message. The component is already quite huge and slow as it is now.